airflow/cli/cli_parser.py

```diff
@@ -626,6 +626,20 @@ def positive_int(value):
     ),
     action="store_true",
     default=False)
+ARG_CONN_IMPORT = Arg(
+    ('file',),
+    help='Import connections from a file. Acceptable file formats .json, .yaml, .env',
+    type=str)
```

Comment:
Can you consider to use required parameter here?

Reply:
File is a positional argument, which is by default required.